### PR TITLE
documentation(EJ2-911245): Port the Culture-based number formats Globalization stability support to all the platforms in Spreadsheet

### PR DESCRIPTION
**Pull request by DinakarManickam4212**
Description:

Need to port the culture-based number formats globalization stability support with preview samples to all the platforms in Spreadsheet.

Task Link: https://dev.azure.com/EssentialStudio/Ej2-Web/_workitems/edit/911245

Content Location: https://ej2.syncfusion.com/documentation/spreadsheet/formatting#culture-based-number-formatting

### CODE REVIEW

File: ej2-asp-core-mvc/spreadsheet/formatting.md [hunk 1]
```diff
@@ -123,6 +123,106 @@ Compared to Excel, the date, time, currency, and accounting formats vary across
...
 > The format code should use the default decimal separator (.) and group separator (,).
 
+The code below illustrates how culture-based format codes are mapped to their corresponding number format ID for the `German` culture.
+
+{% if page.publishingplatform == "aspnet-core" %}
+
+```csharp
+
+Dictionary<string, List<object>> localeFormats = new Dictionary<string, List<object>> {
+{ "de", new List<object> {
```
Comment (sridhar-alagirisamy) [inline ej2-asp-core-mvc/spreadsheet/formatting.md:133]: Don't use Dictionary here. directly define only the German locale formats like other platform why defining Dictionary and  adding "de".

File: ej2-asp-core-mvc/spreadsheet/formatting.md [hunk 2]
```diff
@@ -123,6 +123,106 @@ Compared to Excel, the date, time, currency, and accounting formats vary across
...
+    new { id = 20, code = @"hh:mm" },
+    new { id = 21, code = @"hh:mm:ss" },
+    new { id = 22, code = @"dd.MM.yyyy hh:mm" }
+}}
+}
+ViewBag.LocaleFormats = localeFormats;
+
+<script>
+    var localeFormats = @Html.Raw(Json.Serialize(localeFormats));
+    // Mapping culture-based number formats for the "de" culture: The "spreadsheet" parameter is an instance of the spreadsheet component, and the "deLocaleFormats" parameter is an array containing format codes and their corresponding format IDs for the "de" culture.
```
Comment (sridhar-alagirisamy) [inline ej2-asp-core-mvc/spreadsheet/formatting.md:159]: In Comments mentioned as deLocaleFormats but above defined variable name as localeFormats .

File: ej2-asp-core-mvc/spreadsheet/formatting.md [hunk 3]
```diff
@@ -123,6 +123,106 @@ Compared to Excel, the date, time, currency, and accounting formats vary across
...
+    ej.spreadsheet.configureLocalizedFormat(spreadsheet, localeFormats['de']);
```
Comment (sridhar-alagirisamy) [inline ej2-asp-core-mvc/spreadsheet/formatting.md:160]: Use directly the localeFormats as List<object>. Don't added extra "de".

File: ej2-asp-core-mvc/spreadsheet/formatting.md [hunk 4]
```diff
@@ -123,6 +123,106 @@ Compared to Excel, the date, time, currency, and accounting formats vary across
...
+</script>
+```
+{% elsif page.publishingplatform == "aspnet-mvc" %}
+
+```csharp
+
+Dictionary<string, List<object>> localeFormats = new Dictionary<string, List<object>> {
```
Comment (sridhar-alagirisamy) [inline ej2-asp-core-mvc/spreadsheet/formatting.md:167]: Same as above

File: ej2-asp-core-mvc/spreadsheet/formatting.md [hunk 5]
```diff
@@ -123,6 +123,106 @@ Compared to Excel, the date, time, currency, and accounting formats vary across
...
+    new { id = 20, code = @"hh:mm" },
+    new { id = 21, code = @"hh:mm:ss" },
+    new { id = 22, code = @"dd.MM.yyyy hh:mm" }
+}}
+}
+ViewBag.LocaleFormats = localeFormats;
+
+<script>
+    var localeFormats = @Html.Raw(Json.Encode(localeFormats));
+    // Mapping culture-based number formats for the "de" culture: The "spreadsheet" parameter is an instance of the spreadsheet component, and the "deLocaleFormats" parameter is an array containing format codes and their corresponding format IDs for the "de" culture.
```
Comment (sridhar-alagirisamy) [inline ej2-asp-core-mvc/spreadsheet/formatting.md:194]: Same as above.

File: ej2-asp-core-mvc/spreadsheet/formatting.md [hunk 6]
```diff
@@ -123,6 +123,106 @@ Compared to Excel, the date, time, currency, and accounting formats vary across
...
+    ej.spreadsheet.configureLocalizedFormat(spreadsheet, localeFormats['de']);
```
Comment (sridhar-alagirisamy) [inline ej2-asp-core-mvc/spreadsheet/formatting.md:195]: Same as above.

**Review by sridhar-alagirisamy — CHANGES_REQUESTED** [review]

**Review by gobalakrishnans — APPROVED** [review]

**Review by sridhar-alagirisamy — APPROVED** [review]

**Pull request merged**